From 654e0079675aec82db96f8f291f16adb6970da80 Mon Sep 17 00:00:00 2001
From: Yuka Poppe <yuka@netbox.org>
Date: Fri, 30 Mar 2018 20:26:58 +0200
Subject: [PATCH] TRITON-289 "booter menu-lst" and "booter boot-ipxe" commands
 don't work Reviewed by: Cody Peter Mello <cody.mello@joyent.com> Approved by:
 Cody Peter Mello <cody.mello@joyent.com>

---
 lib/booter.js | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/lib/booter.js b/lib/booter.js
index 5fa83eb..f278a21 100644
--- a/lib/booter.js
+++ b/lib/booter.js
@@ -73,6 +73,7 @@ function main() {
     case 'menu-lst':
         bootparams.getBootParams({
             adminUuid: config.adminUuid,
+            tftpRoot: config.tftpRoot,
             cacheDir: config.cache.dir,
             mac: getArg('MAC address'),
             napi: napi,
@@ -83,7 +84,7 @@ function main() {
                 return console.error(err.code + ': ' + err.message);
             }
 
-            return menu.buildMenuLst(res, config.tftpRoot, function (lst) {
+            return menu.buildMenuLst(res, function (lst) {
                 return console.log(lst);
             });
         });
@@ -94,6 +95,7 @@ function main() {
     case 'boot-ipxe':
         bootparams.getBootParams({
             adminUuid: config.adminUuid,
+            tftpRoot: config.tftpRoot,
             cacheDir: config.cache.dir,
             mac: getArg('MAC address'),
             napi: napi,
@@ -104,7 +106,7 @@ function main() {
                 return console.error(err.code + ': ' + err.message);
             }
 
-            menu.buildIpxeCfg(res, config.tftpRoot, function (cfg) {
+            menu.buildIpxeCfg(res, function (cfg) {
                 return console.log(cfg);
             });
         });
-- 
2.21.0

